account.go

```diff
@@ -342,11 +342,14 @@ type Gender string
 
 // Owner is the structure for an account owner.
 type Owner struct {
-	Address      Address              `json:"address" form:"address"`
-	DOB          DOB                  `json:"dob" form:"dob"`
-	First        string               `json:"first_name" form:"first_name"`
-	Last         string               `json:"last_name" form:"last_name"`
-	Verification IdentityVerification `json:"verification" form:"verification"`
+	Address            Address              `json:"address" form:"address"`
+	DOB                DOB                  `json:"dob" form:"dob"`
+	First              string               `json:"first_name" form:"first_name"`
+	Last               string               `json:"last_name" form:"last_name"`
+	MaidenName         string               `json:"maiden_name" form:"maiden_name"`
+	PersonalID         string               `json:"-" form:"personal_id_number"`
+	PersonalIDProvided bool                 `json:"personal_id_number_provided" form:"-"`
```

Comment:
Would you mind:

(1) Change `PersonalID` to `PersonalIDNumber` — we're trying to move away from the arbitrary shorthands in `stripe-go`.
(2) Likewise change `PersonalIDProvided` to `PersonalIDNumberProvided`.